### PR TITLE
docs(node): Add recipe for deploying serverless functions to Netlify

### PR DESCRIPTION
**Pull request by ndcunningham**
Adds a new entry in the `Deployment` under `Recipes` with a guide for deploying Node Serverless functions to Netlify.

https://nx-dev-git-fork-ndcunningham-docs-node-serverless-f-ea4c98-nrwl.vercel.app/

### CODE REVIEW

File: docs/shared/recipes/deployment/node-serverless-functions-netlify.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Deploying Node serverless functions to Netlify
+
+Deploying node serverless functions to Netlify involes a few steps:
+
+## New workspaces
+
+For new workspaces you can create a Nx workspace with serverless function with one command:
+
+```bash
```
Comment (bcabanes) [inline docs/shared/recipes/deployment/node-serverless-functions-netlify.md:9]: Please use `shell` instead.

**Review by bcabanes — CHANGES_REQUESTED** [review]

File: docs/map.json [hunk 2]
```diff
@@ -1041,6 +1041,12 @@
               "id": "node-server-fly-io",
               "tags": ["deployment", "node"],
               "file": "shared/recipes/deployment/node-server-fly-io"
+            },
+            {
+              "name": "Deploying Node serverless functions to Netlify",
```
Comment (jaysoo) [inline docs/map.json:1046]: Use `Node.js` over `Node`, and make sure it consistent in the recipe.

File: docs/shared/recipes/deployment/node-serverless-functions-netlify.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Deploying Node serverless functions to Netlify
+
+Deploying node serverless functions to Netlify involes a few steps:
+
+## New workspaces
```
Comment (jaysoo) [inline docs/shared/recipes/deployment/node-serverless-functions-netlify.md:5]: Rename to `Creating the project` for consistency with the other recipe.

File: docs/shared/recipes/deployment/node-serverless-functions-netlify.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+For new workspaces you can create a Nx workspace with serverless function with one command:
+
+```bash
+npx create-nx-workspace@latest my-functions \
+--preset=@nrwl/netlify \
+--site=my-site \ # Site ID or name to deploy the functions
+```
+
+## Existing
```
Comment (jaysoo) [inline docs/shared/recipes/deployment/node-serverless-functions-netlify.md:15]: Update as `### Configure existing projects` for consistency.

**Review by jaysoo — APPROVED** [review]

**Review by bcabanes — APPROVED** [review]

**Pull request merged**